src/Tools/dotnet-monitor/ConfigurationTokenParser.cs

```diff
@@ -35,15 +39,21 @@ internal sealed class ConfigurationTokenParser
         public const string Separator = ".";
 
         private const string ProcessInfoReference = "Process";
+        private const string ActionReference = "Action";
+
         private const string RuntimeId = "RuntimeId";
         private const string ProcessId = "ProcessId";
         private const string ProcessName = "Name";
         private const string CommandLine = "CommandLine";
+        private const string Hostname = "Hostname";
+        private const string UnixTime = "UnixTime";
 
         public static readonly string RuntimeIdReference = CreateTokenReference(ProcessInfoReference, RuntimeId);
         public static readonly string ProcessIdReference = CreateTokenReference(ProcessInfoReference, ProcessId);
         public static readonly string ProcessNameReference = CreateTokenReference(ProcessInfoReference, ProcessName);
         public static readonly string CommandLineReference = CreateTokenReference(ProcessInfoReference, CommandLine);
+        public static readonly string HostnameReference = CreateTokenReference(ProcessInfoReference, Hostname);
+        public static readonly string UnixTimeReference = CreateTokenReference(ActionReference, UnixTime);
```

Comment:
I think we should use a different category for Timestamp than Action. These typically refer to other actions in a chain.

Reply:
I am now using `Monitor` as the category for both the time and the host name, as they are both relative to `dotnet-monitor` (and its host). That is also why I added them to the new `HostInfo` class. Let me know how you feel about this.